components/pkg-export-kubernetes/src/manifestjson.rs

```diff
@@ -60,6 +72,12 @@ impl ManifestJson {
         }
     }
 
+    /// Convert into a string. The returned string is the final manifest YAML file content, ready
+    /// for consumption by your Kubernetes cluster.
+    ///
+    /// # Errors
+    ///
+    /// * rendering from the template fails (unlikely)
```

Comment:
Unlikely is not very informative. Maybe specify when exactly it can happen or remove it altogether?

Reply:
It tells the reader that they shouldn't be too scared of just unwrap()ing the result. Since the template is provided by us (from a separate file in the repository), the only reason it can possibly fail is a programmer error (messing up the template) or the cmdline options not properly checked (sill a programmer error). Not sure how to write that here.

Reply:
"Rendering from the template fails. This can only happen if the template has a syntax error or the command-line arguments are inconsistent" - or something like that.

Reply:
Actually since it's likely going to be caused by a programming error, I wonder if this method should should just return a String and use expect() for errors.